pmg/templates/committee_detail.html

```diff
@@ -16,73 +16,98 @@ <h3 class="collapse-link"><span class="fa fa-caret-right"></span> <a href="#">Ab
         </div>
       {% endif %}
 
-      <h3><span class="fa fa-flash"></span> Committee Meetings</h3>
-      {%- set per_page = 20 -%}
-      {%- set num_pages = ((committee.events | length) / per_page) | int -%}
-      {%- if (committee.events | length) % per_page != 0 %}
-        {% set num_pages = num_pages + 1 %}
-      {% endif -%}
+      <nav class="c-m-nav" id="c-m-nav">
+        <ul class="nav nav-tabs">
+          <li class="active"><a href="#cm" data-toggle="tab">Committee meetings</a></li>
+          <li><a href="#cc" data-toggle="tab">Calls for comment</a></li>
+          <li><a href="#tr" data-toggle="tab">Tabled reports</a></li>
+          <li><a href="#qr" data-toggle="tab">Questions/replies</a></li>
+        </ul>
+      </nav>
+
+      <div class="c-m-mobile">
+        <nav class="c-m-nav-mobile" id="c-m-nav-mobile">
+          <div class="dropdown">
+            <button type="button" class="btn btn-default" data-toggle="dropdown">Sections <span class="carat"></span></button>
```

Comment:
How horrible does it look if we just use a <select> element here? Right now, the dropdown doesn't change to indicate what is the currently chosen selection. If we use a <select> and it looks ok, it'd do that automatically.